docs/competitions.md

```diff
@@ -1,24 +1,6 @@
 # Competitions
 
-## Competition 1: SN9_MODEL
-
-Competition 1 is the OG competition for the finetuning subnet. 
-
-### Goal
-
-The purpose of this competition is to finetune the top models from the [pretraining subnet](https://www.macrocosmos.ai/sn9) to produce a chat bot.
```

Comment:
Rather than deleting this, can we move it to a "deprecated" or "completed competitions" section?